Fitter/python/AnomalousCouplingEFTNegative.py

```diff
@@ -289,24 +178,27 @@ def doParametersOfInterest(self):
     def getYieldScale(self,bin,process):
 
         if any( process.startswith(x) for x in self.sgnl_known):
-            if self.sm_re.search(process): 
-                return "func_sm"
+            if self.sm_re.search(process):
+                match=self.sm_re.search(process)
+                return "func_%s_sm"%(match.group('proc'))
             elif self.lin_re.search(process): 
                 match=self.lin_re.search(process)
-                return "func_sm_linear_quadratic_"+ match.group('c1')
+                return "func_%s_sm_linear_quadratic_"%(match.group('proc'))+ match.group('c1')
```

Comment:
It looks like `proc` is defined _below_ this statement? Is this a bug?

Reply:
not sure I understand, here we are calling `'proc'`, not `proc`...

Reply:
Oh I looked passed the quotes!